samples/LibuvWithNonAllocatingFormatters/LibuvWithNonAllocatingFormatters.csproj

```diff
@@ -6,7 +6,6 @@
     <OutputType>Exe</OutputType>
```

Comment:
Note that this sample was removed from the solution as it no longer builds (we removed System.Net.Libuv). What do we plan to do with this?